brain/sw/sync-idf.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+IDF_REV=138c941fad431d3ef2ccbe3eaabf08b96cdc4d0b
+
+if [ "x${IDF_PATH}x" == "xx" ]; then
+  echo 'IDF_PATH isn'\''t set. Fix it!'
+  exit 1
+fi
+
+set -x
+
+cd $IDF_PATH
+git fetch
+git checkout --recurse-submodules $IDF_REV
```

Comment:
This also needs to run the new setup script, which might not be valid for all users (although it really should be). And if you haven't removed an old installation of the xtensa tools I think you're going to get conflicts.

Should be more info in the readme about this, and I'd say that text should also be updated if a new tool like this script is being introduced.